raiden/tests/utils/factories.py

```diff
@@ -185,33 +172,71 @@ def make_channel(
         settle_transaction=settle_transaction,
     )
 
-    return channel_state
+
+def make_transfer_description(
+        payment_network_identifier: typing.PaymentNetworkID = EMPTY,
+        payment_identifier: typing.PaymentID = EMPTY,
+        amount: typing.TokenAmount = EMPTY,
+        token_network: typing.TokenNetworkID = EMPTY,
+        initiator: typing.InitiatorAddress = EMPTY,
+        target: typing.TargetAddress = EMPTY,
+        secret: typing.Secret = EMPTY,
+) -> TransferDescriptionWithSecretState:
+    payment_network_identifier = if_empty(
+        payment_network_identifier, UNIT_PAYMENT_NETWORK_IDENTIFIER)
+    payment_identifier = if_empty(payment_identifier, UNIT_TRANSFER_IDENTIFIER)
+    amount = if_empty(amount, UNIT_TRANSFER_AMOUNT)
+    token_network = if_empty(token_network, UNIT_TOKEN_NETWORK_ADDRESS)
+    initiator = if_empty(initiator, UNIT_TRANSFER_INITIATOR)
+    target = if_empty(target, UNIT_TRANSFER_TARGET)
+    secret = if_empty(secret, random_secret())
+
+    return TransferDescriptionWithSecretState(
+        payment_network_identifier=payment_network_identifier,
+        payment_identifier=payment_identifier,
+        amount=amount,
+        token_network_identifier=token_network,
+        initiator=initiator,
+        target=target,
+        secret=secret,
+    )
 
 
 def make_transfer(
-        amount,
-        initiator,
-        target,
-        expiration,
-        secret,
-        identifier=1,
-        nonce=1,
-        transferred_amount=0,
-        locked_amount=None,
-        token_network_identifier=UNIT_TOKEN_NETWORK_ADDRESS,
-        channel_identifier=UNIT_CHANNEL_ID,
-        locksroot=None,
-        token=UNIT_TOKEN_ADDRESS,
+        amount: typing.TokenAmount = EMPTY,
+        initiator: typing.InitiatorAddress = EMPTY,
+        target: typing.TargetAddress = EMPTY,
+        expiration: typing.BlockExpiration = EMPTY,
+        secret: typing.Secret = EMPTY,
+        identifier: typing.PaymentID = EMPTY,
+        nonce: typing.Nonce = EMPTY,
+        transferred_amount: typing.TokenAmount = EMPTY,
+        locked_amount: typing.TokenAmount = EMPTY,
+        token_network_identifier: typing.TokenNetworkID = EMPTY,
+        channel_identifier: typing.ChannelID = EMPTY,
+        locksroot: typing.Locksroot = EMPTY,
+        token: typing.TargetAddress = EMPTY,
 ):
+    amount = if_empty(amount, UNIT_TRANSFER_AMOUNT)
+    initiator = if_empty(initiator, make_address())
+    target = if_empty(target, make_address())
+    expiration = if_empty(expiration, UNIT_REVEAL_TIMEOUT)
+    secret = if_empty(secret, make_secret())
+    identifier = if_empty(identifier, 1)
```

Comment:
Why not also generate a random identifier here too?

Reply:
As I said above perhaps we should see if they do and then correct them to not expect a specific identifier.